extensions/chat/common/client.mjs

```diff
@@ -95,26 +95,46 @@ Usage: !/<Command> <arg1> <arg2> ...
         document.getElementById('message_history').innerHTML += '<span>Invalid Command</span><br>';
         this.listCommand();
       }
-    };
+    }
```

Comment:
Not sure if this is the correct style. Will double check later.

Reply:
sorry for no PR description
all fixed by eslint --fix